scripts/update_e2e_coverage.py

```diff
@@ -101,6 +110,34 @@ def parse_arguments():
   return parsed_args
 
 
+def parse_test_name(test_name, test_suite):
+  """Splits a test name into a dictionary with its source file and backend."""
+  test_name = test_name.split("__")
```

Comment:
Changing this and leaving the variable name the same can be confusing. And it looks like you actually use `test_name` below in an error message where it should be the original name. How about `test_name_parts` instead for this?

Reply:
Done.